pkgs/servers/code-server/default.nix

```diff
@@ -220,6 +230,6 @@ in stdenv.mkDerivation rec {
     homepage = "https://github.com/cdr/code-server";
     license = licenses.mit;
     maintainers = with maintainers; [ offline ];
-    platforms = ["x86_64-linux"];
+    platforms = [ "x86_64-linux" "x86_64-darwin" ];
```

Comment:
👍  It looks like aarch64 hasn't been in `platforms` so far. I'll try adding it, but if the build fails, I may take out `aarch64` from the yarn package hashes, since I don't currently have a way to work on `aarch64` packages.

Reply:
Looks like it built successfully, excellent